io.edgehog.devicemanager.RemoteTerminal.json

```
@@ -0,0 +1,26 @@
+{
+    "interface_name": "io.edgehog.devicemanager.RemoteTerminal",
+    "version_major": 0,
+    "version_minor": 1,
+    "type": "datastream",
+    "ownership": "server",
+    "aggregation": "object",
+    "description": "Configuration to open a remote terminal from a device to a certain host",
+    "mappings": [
+        {
+            "endpoint": "/terminal/host",
+            "type": "string",
+            "description": "Send the IP address or host name the device must connect to"
```

Add a database retention policy to define that a data should expire from the database after a given interval.
`31556952` -> `~ 1 year`
```suggestion
            "description": "Send the IP address or host name the device must connect to"
            "database_retention_policy": "use_ttl",
            "database_retention_ttl": 31556952,
```